From 4794ccc075de763f0c1aa6157158510f820ead7b Mon Sep 17 00:00:00 2001
From: thau0x01 <thau0x01@hackware.tech>
Date: Tue, 10 Jan 2023 17:27:05 -0300
Subject: [PATCH 05/15] try fix patch

---
 lib/base/rpc.vala | 16 ++++++++--------
 1 file changed, 8 insertions(+), 8 deletions(-)

diff --git a/lib/base/rpc.vala b/lib/base/rpc.vala
index 02602abf..e09e745c 100644
--- a/lib/base/rpc.vala
+++ b/lib/base/rpc.vala
@@ -15,12 +15,12 @@ namespace Frida {
 			string request_id = Uuid.string_random ();
 
 			var request = new Json.Builder ();
-			request
-				.begin_array ()
-				.add_string_value ((string) GLib.Base64.decode("ZnJpZGE6cnBj="))
-				.add_string_value (request_id)
-				.add_string_value ("call")
-				.add_string_value (method)
+				request
+					.begin_array ()
+					.add_string_value ("frida:rpc")
+					.add_string_value (request_id)
+					.add_string_value ("call")
+					.add_string_value (method)
 				.begin_array ();
 			foreach (var arg in args)
 				request.add_value (arg);
@@ -70,7 +70,7 @@ namespace Frida {
 		}
 
 		public bool try_handle_message (string json) {
-			if (json.index_of ((string) GLib.Base64.decode("ImZyaWRhOnJwYyI=")) == -1)
+			if (json.index_of ("\"frida:rpc\"") == -1)
 				return false;
 
 			var parser = new Json.Parser ();
@@ -99,7 +99,7 @@ namespace Frida {
 				return false;
 
 			string? type = rpc_message.get_element (0).get_string ();
-			if (type == null || type != (string) GLib.Base64.decode("ZnJpZGE6cnBj="))
+			if (type == null || type != "frida:rpc")
 				return false;
 
 			var request_id_value = rpc_message.get_element (1);
-- 
2.39.0

